public/app/features/explore/Logs.tsx

```diff
@@ -142,7 +141,7 @@ export default class Logs extends PureComponent<Props, State> {
 
   onToggleLogLevel = (rawLevel: string, hiddenRawLevels: Set<string>) => {
```

Comment:
Minor: can ``rawLevel: string`` be removed from function signature?

Reply:
I don't want to remove it now as this is a part of `Graph` component API and I'm not sure what is the cost of removing it.